### PR TITLE
Add extra data in transfer in completed event

### PR DESCRIPTION
**Pull request by gius80**
Adding some extra data in `Domain\Transfer\In\Completed` event.

#### Testing
```
> composer install
> ./vendor/bin/phpunit -c ./test/phpunit.xml
```

### CODE REVIEW

File: lib/event/renewable-until-trait.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, see https://www.gnu.org/licenses.
+ */
+
+namespace Automattic\Domain_Services_Client\Event;
+
+use Automattic\Domain_Services_Client\{Helper};
+
+/**
+ * Trait that adds the `renewable_until` method to an event
```
Comment (leonardost) [inline lib/event/renewable-until-trait.php]:
```suggestion
 * Trait that adds the `get_renewable_until` method to an event
```
Nit

File: lib/event/domain/register/success.php [hunk 2]
```diff
@@ -62,50 +67,6 @@ public function get_domain_status(): Entity\Epp_Status_Codes {
...
-	 * Gets the date when the domain will be suspended if the contact information is not verified
-	 *
-	 * @return null|\DateTimeInterface
-	 */
-	public function get_unverified_contact_suspension_date(): ?\DateTimeInterface {
-		$unverified_contact_suspension_date = $this->get_data_by_key( 'event_data.unverified_contact_suspension_date' );
-
-		return null === $unverified_contact_suspension_date ? null : Helper\Date_Time::createImmutable( $unverified_contact_suspension_date );
-	}
-
```
Comment (leonardost) [inline lib/event/domain/register/success.php:108]: Should we remove those methods from the `Domain/Register/Success` event? These data are still returned from the `Domain/Register` command, aren't they?

**Review by leonardost — APPROVED** [review]
LGTM! There's just one comment in `lib/event/domain/register/success.php` that I think should be checked before deploying.

File: lib/event/domain/register/success.php (hunk 2 above)
Comment (gius80) [reply lib/event/domain/register/success.php:108]: @leonardost I removed those methods since I moved them to treats :)

**Pull request merged**